composer.json

```diff
@@ -29,7 +29,9 @@
     },
     "require-dev": {
         "sensio/generator-bundle": "~2.3",
-        "phpunit/phpunit": "~4.3"
+        "phpunit/phpunit": "~4.3",
+        "ongr/ongr-strict-standard": "~1.0-beta",
+        "squizlabs/php_codesniffer": "~1"
```

Comment:
It will not work with version 2. Please, copy from here: https://github.com/ongr-io/ConnectionsBundle/blob/5a9eaff00a9d7c436b0d63bbd06e651de29280a5/composer.json#L22